#
# missing <cmath> for std::abs() variants.  This is fixed upstream in 3.5.5 
# https://github.com/wxWidgets/wxWidgets/commit/9ce192d417eb9eb614bcf8510e91dac318706249
#
diff -u -r wxWidgets-3.0.2/src/stc/scintilla/src/Editor.cxx wxWidgets-3.0.2.new/src/stc/scintilla/src/Editor.cxx
--- wxWidgets-3.0.2/src/stc/scintilla/src/Editor.cxx	2014-10-06 14:33:44.000000000 -0700
+++ wxWidgets-3.0.2.new/src/stc/scintilla/src/Editor.cxx	2016-09-20 08:00:30.896632125 -0700
@@ -11,6 +11,7 @@
 #include <ctype.h>
 #include <assert.h>
 
+#include <cmath>
 #include <string>
 #include <vector>
 #include <map>
